app/controllers/AddressLookup.scala

```diff
@@ -25,7 +26,17 @@ trait AddressLookup extends PropertyLinkingController {
   val addresses = Wiring().addresses
 
   def findByPostcode(postcode: String) = Action.async { implicit request =>
-    addresses.findByPostcode(postcode) map { res => Ok(Json.toJson(res)) }
+    if (postcode.trim.contains(" ")) {
```

Comment:
Would search by post code prefix be so general as to be not useful? It would usually result in 100s of properties and that wouldn't be easy to choose from.